events/Tribals.txt

```diff
@@ -0,0 +1,485 @@
+		remove_province_modifier = military_hunting_tribals
+		
+		owner = {
+			random_list = {
+				5 = { treasury = 5000 }
+				10 = { treasury = 10000 }
+				4 = { treasury = 15000 }
+				1 = { years_of_research = 0.10 }
+			}
+		}
+	}
+}
+
+# Abandoned Tribal Settlement Found - REMOVE
+province_event = {
+	id = 5002
+	title = "Abandoned Settlement Found"
+	desc = "EVTDESC5002"
+	
+	trigger = {
+		has_province_modifier = military_hunting_tribals
+	}
+	
+	mean_time_to_happen = {
+		months = 6
+		
+		modifier = {
+			factor = 0.5
+			units_in_province = 1
+		}
+	}
+	
+	option = {
+		name = "How are they one step ahead of us?"
+		
+		owner = {
+			random_list = {
+				5 = { treasury = 2500 }
+				10 = { treasury = 5000 }
+				4 = { treasury = 7500 }
+				1 = { years_of_research = 0.05 }
+			}
+		}
+		
+		remove_province_modifier = military_hunting_tribals
+	}
+}
+
+# Tribal Settlement Found and Enslaved - REMOVE
+province_event = {
+	id = 5003
+	title = "Tribal Settlement Found"
+	desc = "EVTDESC5003"
+	
+	trigger = {
+		has_province_modifier = military_hunting_tribals_slavery
+	}
+	
+	mean_time_to_happen = {
+		months = 6
+		
+		modifier = {
+			factor = 0.5
+			units_in_province = 1
+		}
+	}
+	
+	option = {
+		name = "Good"
+		
+		# Kill those who resist
+		any_pop = {
+			limit = {
+				type = tribals
+			}
+			
+			reduce_pop = 0.9
+		}
+		
+		# Enslave the remaining tribals
+		any_pop = {
+			limit = {
+				type = tribals
+			}
+			pop_type = slaves
+		}
+		
+		# Some of the militia died
+		any_pop = {
+			limit = {
+				type = soldiers
+			}
+			
+			reduce_pop = 0.98
+		}
+		
+		any_pop = {
+			militancy = -1
+		}
+		
+		remove_province_modifier = tribal_outpost
+		remove_province_modifier = military_hunting_tribals_slavery
+		
+		owner = {
+			random_list = {
+				5 = { treasury = 5000 }
+				10 = { treasury = 10000 }
+				4 = { treasury = 15000 }
+				1 = { years_of_research = 0.10 }
+			}
+		}
+	}
+}
+
+####################################################################################################
+####################################################################################################
+####################################################################################################
+# on_state_conquest - yes slavery - replace event 70000
+## Convert % of non-accepted pops to slaves
+## Convert % of non-accepted pops to tribals
+
+# on_state_conquest - yes non_accepted_slavery - replace event 70007
+## Convert % of non-accepted pops to slaves
+## Convert % of non-accepted pops to tribals
+
+# on_state_conquest - no slavery
+## Convert % of non-accepted pops to tribals
+
+####################################################################################################
+####################################################################################################
+####################################################################################################
+# Reembrace primary culture tribals
```

Comment:
We can definitely do all of the reembracing in a cleanup event on the world bank (either calling a reembrace event, or just updating the pop types)